pwa/src/pages/configurationsExportButton/index.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react"
+import Layout from "../../components/common/layout";
+import ConfigurationsExportButton from "../../components/configurationsExportButton/ConfigurationsExportButton";
+
+const IndexPage = () => {
+  return (
+    <Layout title={"ConfigurationsExportButton"} subtext={"An overview of your endpoints objects"}>
+      <title>Gateway - ConfigurationsExportButton</title>
```

Comment:
This page is just `Configurations` and not `ConfigurationsExportButton` therefore, the title should be changed.